code/modules/spacepods/equipment.dm

```diff
@@ -49,23 +49,35 @@
 	var/obj/item/device/spacepod_equipment/weaponry/weapon_system // weapons system
 	var/obj/item/device/spacepod_equipment/misc/misc_system // misc system
 	var/obj/item/device/spacepod_equipment/cargo/cargo_system // cargo system
+	var/obj/item/device/spacepod_equipment/cargo/sec_cargo_system // secondary cargo system
 	var/obj/item/device/spacepod_equipment/lock/lock_system // lock system
-	//var/obj/item/device/spacepod_equipment/engine/engine_system // engine system
-	//var/obj/item/device/spacepod_equipment/shield/shield_system // shielding system
 
 /datum/spacepod/equipment/New(var/obj/spacepod/SP)
 	..()
 	if(istype(SP))
 		my_atom = SP
 
+/datum/spacepod/equipment/proc/get_list()
+	var/list/stuff
+	for(var/obj/I in vars)
```

Comment:
I looped over vars because the datum only had equipment, but that solution certainly doesn't scale well so your suggestion makes sense and is actually really easy to implement now that I've condensed equipment additions and removal to single functions